posthog/settings/sentry.py

```diff
@@ -158,15 +156,17 @@ def sentry_init() -> None:
                 DjangoIntegration(),
                 CeleryIntegration(),
                 RedisIntegration(),
-                sentry_logging,
+                ClickhouseDriverIntegration(),
+                LoggingIntegration(level=sentry_logging_level, event_level=None),
             ],
-            request_bodies="always" if send_pii else "never",
+            max_request_body_size="always" if send_pii else "never",
+            max_value_length=8192,  # Increased from the default of 1024 to capture SQL statements in full
             sample_rate=1.0,
-            # Configures the sample rate for error events, in the range of 0.0 to 1.0 (default).
-            # If set to 0.1 only 10% of error events will be sent. Events are picked randomly.
+            # Samplers return True if event should be sent, False if it should be dropped,
+            # or a float between 0 and 1 to send at that specific rate randomly
             traces_sampler=traces_sampler,
+            error_sampler=error_sampler,
```

Comment:
just to be sure this is intentional
`before_send_transaction` and `traces_sampler` are only about transactions.
`before_send` and probably `error_sampler` only for errors.
since you removed one of them, maybe more transactions will be sent?

Reply:
Hmm, good catch, now that you say I remembered this was purely about transactions. Will revert this part